scripts/stitching/stitching.py

```diff
@@ -0,0 +1,325 @@
+""" Script for creating a stitched ROI MIBItiff from a folder of FOVs.
```

Comment:
All methods in this file need a docstring explaining what the function does and what each argument is: https://peps.python.org/pep-0257/#multi-line-docstrings